lib/ui/home.dart

```diff
@@ -47,12 +49,15 @@ class _HomeState extends State<Home> with WidgetsBindingObserver {
     }
   }
 
+
   final List<Widget> _children = [
     PlotList(),
     HomeProfitLossChild(),
     ArticleList(),
     HomeCommunityChild(),
-    PlaygroundView(widgetList: [], appBarColor: Color(0xFF9CBD3A),),
+    PlaygroundView(widgetList: [
+      ActionSheet(viewModel: MockProfitLossListViewModel.build(), style: ActionSheetStyle.defaultStyle())
```

Comment:
maybe we should move this to a another file and add the final widgetList here to avoid having the large widgetlist here